DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/BulkUploadController.cs

```diff
@@ -147,6 +147,10 @@ public IActionResult UploadComplete()
         [Route("WelcomeEmail")]
         public IActionResult WelcomeEmail()
         {
+            if (!TempData.Any())
+            {
+                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 401 });

```

Comment:
Again the same here. we need 410 error instead of 401.

Reply:
Done